os/board/common/crashdump.c

```diff
@@ -26,7 +26,6 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <chip.h>
```

Comment:
Could you confirm this does not cause build break on artik, imxrt and mtk?

Reply:
Verified successful build for the following configurations after enabling ramdump support:

# imxrt1020-evk/loadable_elf_apps
# imxrt1050-evk/loadable_elf_apps
# mt7686_hdk/loadable_apps
# artik053/tc